packages/RNTester/metro.config.js

```diff
@@ -6,7 +6,7 @@
  */
 
 const path = require('path');
-const reactNativePath = path.resolve('../..');
+const reactNativePath = path.resolve('./node_modules/react-native');
```

Comment:
Hey @chirag-singhal! This is just a hack to make RNTester/Detox tests work on our machine locally. This should not be committed.

Reply:
@anku255 Done :+1: